src/task.js

```diff
@@ -0,0 +1,27 @@
+class TASK {
+  constructor() {
+    this.taskList = [];
+  }
+
+  static fetch() {
+    return JSON.parse(localStorage.getItem('task')) || [];
+  }
+
+  static updateStorage(updatedTask) {
+    localStorage.setItem('task', JSON.stringify(updatedTask));
+  }
+
+  add(task) {
+    this.taskList = TASK.fetch();
+    this.taskList.push(task);
+    TASK.updateStorage(this.taskList);
+  }
+
+  updateIndex() {
+    this.taskList.forEach((item, id) => {
+      item.index = id + 1;
+    });
+  }
```

Comment:
I see you used here, none ES6 JS syntax, as much as the code is functional, for better readability, you could consider refactoring this whole section to ES6 syntax. This will help make your code lighter and more readable than it is now.

Reply:
Thank you for your thorough review